packages/ra-auth-cognito/src/RequestNewPasswordForm.tsx

```diff
@@ -0,0 +1,87 @@
+import {
+    Button,
+    CardContent,
+    CircularProgress,
+    Typography,
+} from '@mui/material';
+import clsx from 'clsx';
+import * as React from 'react';
+import {
+    Form,
+    LoginFormClasses,
+    TextInput,
+    required,
+    useTranslate,
+} from 'react-admin';
+import type { SubmitHandler } from 'react-hook-form';
+import type { FormData } from './useCognitoLogin';
+import { validatePasswordsMatch } from './validatePasswordsMatch';
+
+export type RequestNewPasswordFormProps = {
+    className?: string;
+    submit: SubmitHandler<FormData>;
+    isLoading?: boolean;
+};
+
+export const RequestNewPasswordForm = ({
+    className,
+    submit,
+    isLoading,
+}: RequestNewPasswordFormProps) => {
+    const translate = useTranslate();
+
+    return (
+        <Form
+            onSubmit={submit}
+            mode="onChange"
+            noValidate
+            className={clsx('RaLoginForm-root', className)}
+        >
+            <CardContent className={LoginFormClasses.content}>
+                <Typography>
+                    {translate('ra.auth.require_new_password', {
+                        _: 'Please enter a new password',
+                    })}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** This key `ra.auth.require_new_password‏` doesn't match anything, neither here nor in react-admin. We probably didn't see it before because it provides a default translation. 

I agree it's not the responsibility of this PR, but since we are introducing language packages, I think it's a good opportunity to add this translation key in them! Wdyt?